content/docs/command-reference/import-url.md

```diff
@@ -51,8 +51,8 @@ corresponding local path in the <abbr>workspace</abbr>. It records enough
 metadata about the imported data to enable DVC efficiently determining whether
 the local copy is out of date.
 
-`dvc repro` doesn't check and/or update generated `.dvc` files, use `dvc update`
-on them to bring the import up to date from the external data source.
+`dvc repro` doesn't check or update generated `.dvc` files, use `dvc update` on
```

Comment:
```suggestion
Note that `dvc repro` doesn't check or update import `.dvc` files, use `dvc update` on
```
Can you please apply this and format the paragraph? A bit out of scope but it relates to something else you added...

Reply:
Also, in line 48 can you change "In such a `.dvc`" for "In such an import `.dvc`"? Thanks